cognite_toolkit/_cdf_tk/client/data_classes/migration.py

```diff
@@ -223,13 +223,90 @@ def __init__(
         self.view_id = view_id
         self.mapping = mapping
 
-    def _dump_properties(self) -> dict[str, Any]:
-        """Dump the properties of the ViewSourceApply."""
-        return {
-            "resourceType": self.resource_type,
-            "viewId": self.view_id.dump(),
-            "mapping": self.mapping.dump(),
-        }
+    def dump(self, camel_case: bool = True, context: Literal["api", "local"] = "api") -> dict[str, Any]:
```

Comment:
We have to update the dump method to support both dumping to the API format (a node), and the YAML format we use locally in Toolkit.